src/BenchmarkDotNet.Core/Portability/HyperV.cs

```diff
@@ -0,0 +1,13 @@
+namespace BenchmarkDotNet.Portability
+{
+    public class HyperV : VirtualMachineHypervisor
+    {
+        public override string Name => "Hyper-V";
+
+        public override bool IsVirtualMachine(string manufacturer, string model)
+        {
+            return manufacturer != null && model != null && 
```

Comment:
LGTM.
Also please use `[NotNull]` and `[CanBeNull]` attributes where it's possible.